### DISCUSSION TITLE
HTTPS for Kafka UI

### DISCUSSION THREAD

**Discussion by cassanellicarlo** [provectus/kafka-ui#2384] [Q&A] [answered]
Hello,
I wanted to ask if it's possible to enable https for the kafka ui.

To be able to access https://localhost:8080 instead of http://localhost:8080

Thank you

**Comment by Haarolean** [comment, marked as answer]
Hi, I really suggest using a proxy for that (nginx, traefik, etc.).
But if you really would like to do it on app's side: check [this](https://github.com/provectus/kafka-ui/issues/479#issuecomment-1001041589) out.
